src/app.module.ts

```diff
@@ -37,6 +37,12 @@ const mocks = {
       driver: ApolloDriver,
       playground: false,
       debug: true,
+      cors: {
+        // TODO: origin should be moved into configModule - not sure regex is valid!
+        // https://www.debuggex.com/r/XTrC-yG2Yeq2_sAm
+        origin: /https?:\/\/[a-z0-9\-_]+-fullstacks\.vercel\.app/,
+        credentials: true,
```

Comment:
cause we don't use cookie credentials could be false.

Reply:
Based on [Access-Control-Allow-Credentials](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Credentials) explanations
> Credentials are cookies, authorization headers, or TLS client certificates.

Should the authorization header send cross requests? if there is no auth header `credentials` is useless

Reply:
Oh, i didn't know that (which Authorization header counts too)